python/doc/examples/meta_modeling/general_purpose_metamodels/plot_expert_mixture.py

```diff
@@ -3,35 +3,33 @@
 ==================
```

Comment:
As a side note, you might be interested by the unit test provided in [otmixmod/t_MixtureFactory_expert.py](https://github.com/openturns/otmixmod/blob/master/python/test/t_MixtureFactory_expert.py).